# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/gc/regress-1711413.js
# Commit: 676659d41d31
# Full Hash: 676659d41d31a9ea253b4bedc0331053c264995d
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-06-01 21:33:58
# Description:
#   Bug 1711413 - Fix marking of nuked weakmap key delegates r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D115750
# ==============================================================================

diff -r d2e43a7b0526 -r 676659d41d31 js/src/jit-test/tests/gc/regress-1711413.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/gc/regress-1711413.js	Tue Jun 01 17:16:13 2021 +0000
@@ -0,0 +1,6 @@
+// Required --no-threads to crash.
+
+enableShellAllocationMetadataBuilder();
+newGlobal({newCompartment:true});
+gcslice(1);
+nukeAllCCWs();
